docs/public-networks/how-to/pre-merge-history-expiry.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Prune pre-merge history
+sidebar_position: 9
+description: Reduce the size of your database by removing pre-merge blocks from your blockchain
+tags:
+  - public networks
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Node operators using [Snap sync](../concepts/node-sync.md#snap-synchronization) can significantly reduce
+disk usage by removing [pre-merge](https://ethereum.org/en/roadmap/merge/) Proof of Work (PoW)
+block data from the local database.
+
+Besu can prune all pre-merge blocks and associated transaction receipts, leaving only headers and
+the genesis block.
+
+:::warning
+
+Besu does not currently provide a way to manually import pre-merge block data after pruning.
+If you need to restore the full pre-merge history, you can revert to the old sync method and download
+all blocks from peers by setting
+[`--snapsync-synchronizer-pre-checkpoint-headers-only-enabled=false`](../reference/cli/options.md#snapsync-synchronizer-pre-checkpoint-headers-only-enabled).
+
+Support for Era1 sync for archive nodes is currently in progress.
+
+:::
+
+Besu provides multiple options to prune the historical blockchain data:
+- [Offline pruning](#offline-pruning) to prune data on a stopped Besu instance.
+- [Online pruning](#online-pruning) to prune data on a running Besu instance.
+- [Sync Besu](#sync-without-pre-merge-blocks) using Snap sync, which by default prunes the historical
+    blockchain data.
+
+## Offline pruning
+
+The easiest and fastest option for pruning pre-merge blocks is to perform an offline prune.
```

Comment:
```suggestion
The fastest option for pruning pre-merge blocks is to perform an offline prune. It won't prune as much data as a full resync.
```

Reply:
Easiest is probably resyncing, but that has the most downtime.
This is the most number of steps so not easiest.